packages/algob/src/lib/deployer.ts

```diff
@@ -85,13 +87,12 @@ export class AlgobDeployerImpl implements AlgobDeployer {
     return this.cpData.precedingCP[this.networkName].asa[name];
   }
 
-  async deployASC (name: string, source: string, account: Account): Promise<ASCInfo> {
+  async deployASC (name: string, scParams: Object, flags: ASCDeploymentFlags,
+    payFlags: ASCPaymentFlags): Promise<ASCInfo> {
+    const creator = flags.funder;
```

Comment:
You define a variable just to use it in next line and then you also pass `flags` too which has this parameter.

Reply:
deployASA was used for reference , that has same structure

Reply:
But you pass the params twice. You don't need creator parameter in `this.algoOp.deployASC(name, scParams, flags, creator)`

Reply:
ok